src/app/rpmostree-builtin-testutils.cxx

```diff
@@ -136,3 +146,18 @@ rpmostree_testutils_builtin_inject_pkglist (int argc, char **argv,
   g_print("%s => %s\n", refspec, new_checksum);
   return TRUE;
 }
+
+gboolean
+rpmostree_testutils_builtin_script_shell (int argc, char **argv,
+                                          RpmOstreeCommandInvocation *invocation,
+                                          GCancellable *cancellable, GError **error)
+{
+  const char *rootpath = "/";
+  if (argc > 1)
+    rootpath = argv[1];
+  glnx_autofd int rootfs_dfd = -1;
+  if (!glnx_opendirat (AT_FDCWD, rootpath, TRUE, &rootfs_dfd, error))
+    return FALSE;
+
+  return rpmostree_run_script_in_bwrap_container (rootfs_dfd, NULL, TRUE, "testscript", NULL, "/bin/bash", "exec /bin/bash", NULL, 0, cancellable, error);
```

Comment:
Minor (here and in `rpmostree_run_script_in_bwrap_container`): I like `STDIN_FILENO` instead of `0` in these cases because it parses more quickly when reading that it's not just another optional/default arg value.

Reply:
Hmm, couldn't we pass `NULL` here for `interp` and `script` since they're not actually used and also only allocate a memfd if `script != NULL`?

Reply:
Yep, good calls!  Fixed.